### DISCUSSION TITLE
[graphgym] Custom aggregation functions

### DISCUSSION THREAD

**Discussion by smorad** [pyg-team/pytorch_geometric#3340] [Q&A]
I see that pyg already has a great framework for comparing specific portions of a GNN: https://github.com/snap-stanford/GraphGym#4-user-customization

I don't see a way to compare aggregation functions. Is there a way to compare new aggregation function against `sum` or `mean`?

**Comment by rusty1s** [comment]
You should be able to compare aggregation functions by doing a hyperparameter search over `gnn.agg`, see [here](https://github.com/pyg-team/pytorch_geometric/blob/master/graphgym/grids/pyg/example.txt).
